packages/daf-core/src/graphql/graphql-core.ts

```diff
@@ -113,52 +105,217 @@ function transformFindInput(input: FindInput): FindManyOptions {
       if (item.not === true) {
         where[item.column] = Not(where[item.column])
       }
+      return where
     }
-    result['where'] = where
   }
-  return result
+}
+
+function decorateQB(
+  qb: SelectQueryBuilder<any>,
+  tablename: string,
+  input: FindInput,
+): SelectQueryBuilder<any> {
+  if (input?.skip) qb = qb.skip(input.skip)
+  if (input?.take) qb = qb.take(input.take)
+
+  if (input?.order) {
+    for (const item of input.order) {
+      qb = qb.orderBy(`${escape(tablename)}.${escape(item.column)}`, item.direction)
```

Comment:
No need to use another package for that. You can do the same thing with:
``` typescript
qb = qb.orderBy(`${qb.connection.driver.escape(tablename)}.${qb.connection.driver.escape(item.column)}`, item.direction)
```

Reply:
oh cool, i looked for this, guess I missed it.